src/lib/autocomplete/autocomplete.spec.ts

```diff
@@ -1090,6 +1099,24 @@ describe('MdAutocomplete', () => {
     expect(Math.ceil(parseFloat(overlayPane.style.width))).toEqual(500);
 
   });
+
+  it('should show the panel when the options are initialized later within a component with ' +
+    'OnPush change detection', fakeAsync(() => {
+      let fixture = TestBed.createComponent(AutocompleteWithOnPushDelay);
+
+      fixture.detectChanges();
+      dispatchFakeEvent(fixture.debugElement.query(By.css('input')).nativeElement, 'focus');
+      tick(1000);
+      fixture.detectChanges();
+
+      Promise.resolve().then(() => {
+        let panel = overlayContainerElement.querySelector('.mat-autocomplete-panel') as HTMLElement;
+        let visibleClass = 'mat-autocomplete-visible';
+
+        fixture.detectChanges();
+        expect(panel.classList).toContain(visibleClass, `Expected panel to be visible.`);
```

Comment:
Does this `Promise` ever resolve? Don't you need another `flushMicroTasks` to make it run?